File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciorder/d/FeeReverseType.java
package iih.ci.ord.ciorder.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface FeeReverseType {

    @DmEnumDesc(name="记账后",description="【记账状态】=已记账，冲账类型 = 0")
	public static final Integer NOBLCANCEL=0; //记账后
    @DmEnumDesc(name="销账",description="记账未结算后的销账")
	public static final Integer BLCGCANCEL=1; //销账
    @DmEnumDesc(name="结算后冲账",description="记账并结算后的冲账")
	public static final Integer BLSTREVERSE=2; //结算后冲账
}	

File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciorder/d/FeeReverseType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.ciorder.d
{
   public enum FeeReverseType {
        /// <summary>
        /// 记账后
        /// </summary>
        [Description("【记账状态】=已记账，冲账类型 = 0")]
        NOBLCANCEL =0,
	/// <summary>
	/// 销账
	/// </summary>
    [Description("销账")]
    BLCGCANCEL=1,
	/// <summary>
	/// 结算后冲账
	/// </summary>
    [Description("结算后冲账")]
    BLSTREVERSE=2,
   }
}
